R/source.R

```diff
@@ -251,24 +251,25 @@ invoke_cargo <- function(toolchain, specific_target, dir, profile,
     # rtools_path() returns path to the RTOOLS40_HOME\usr\bin,
     # but we need RTOOLS40_HOME\mingw{arch}\bin, hence the "../.."
     rtools_home <- normalizePath(
-      # `pkgbuild` may return two paths for R < 4.2 if Rtools42 is present
+      # `pkgbuild` may return two paths for R < 4.2 with Rtools40v2
       file.path(pkgbuild::rtools_path()[1], "..", ".."),
       winslash = "/",
       mustWork = TRUE
     )
```

Comment:
Are we sure that the first returned path is the correct one? What if both v40 and v42 are installed on the system?

Reply:
Yes, I'm pretty sure.

https://github.com/r-lib/pkgbuild/blob/a22b6a3014476efba60adae77858d4ff7ebc1deb/R/rtools.R#L37-L43